playwright/tests/responsevalidation.spec.ts

```diff
@@ -0,0 +1,50 @@
+import {test, expect} from '@playwright/test';
+import {partialBody} from '../data/response-body';
+const fullBody = require('../data/users')
```

Comment:
In the long run, we can review test data management ideas as well. Fixtures of Plazwright are something to consider. We can give it a shot at our sync meeting.

Reply:
Did not take care of this in the latest commit, but we can take a look at it later if it is necessary.